assets/alertmanager/alertmanager.yaml

```diff
@@ -59,6 +59,39 @@ spec:
       name: secret-alertmanager-main-tls
     - mountPath: /etc/proxy/secrets
       name: secret-alertmanager-main-proxy
+  - args:
+    - --secure-listen-address=0.0.0.0:9092
+    - --upstream=http://127.0.0.1:9096
+    - --config-file=/etc/kube-rbac-proxy/config.yaml
+    - --tls-cert-file=/etc/tls/private/tls.crt
+    - --tls-private-key-file=/etc/tls/private/tls.key
+    - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+    - --logtostderr=true
+    - --v=10
+    image: quay.io/coreos/kube-rbac-proxy:v0.4.1
+    name: kube-rbac-proxy
+    ports:
+    - containerPort: 9092
+      name: tenancy
+    resources:
+      requests:
+        cpu: 10m
```

Comment:
let's put this down to `1m`. Current measurements show this doesn't need more (at least for requests).